### PR TITLE
[zh-cn] sync 1.27 kubectl.md

### PR DESCRIPTION
**Pull request by asa3311**
sync 1.27 kubectl.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubectl/kubectl.md [hunk 1]
```diff
@@ -528,6 +528,19 @@ Toggles whether calls to `kubectl explain` use the new OpenAPIv3 data source ava
...
+<tr>
+<td colspan="2">KUBECTL_ENABLE_CMD_SHADOW</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+When set to true, external plugins can be used as subcommands for builtin commands if subcommand does not exist. In alpha stage, this feature can only be used for create command(e.g. kubectl create networkpolicy).
+-->
+当设置为 true 时，如果子命令不存在，外部插件可以用作内置命令的子命令。
+在 alpha 阶段，此功能只能用于 create 命令（例如 kubectl create networkpolicy）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/kubectl.md:540]:
```suggestion
此功能处于 alpha 阶段，只能用于 create 命令（例如 kubectl create networkpolicy）。
```

**Comment by windsonsea** [comment]
Run the following commands to trigger the CI again:
```
git reset
git commit --amend --no-edit
git push -f
```

**Pull request merged**